packages/angular_devkit/build_angular/src/webpack/configs/common.ts

```diff
@@ -95,14 +95,30 @@ export async function getCommonConfig(wco: WebpackConfigOptions): Promise<Config
     extraPlugins.push(new ProgressPlugin(platform));
   }
 
+  const localizePackageInitEntryPoint = '@angular/localize/init';
+  if (tsConfig.options.types?.includes(localizePackageInitEntryPoint)) {
```

Comment:
Could we have this check both `@angular/localize/init` and `@angular/localize`? I'd like to expose the global types in the later as well (or eventually instead) since its possible to use `$localize` without the `init` when build localization is enabled.